src/java.base/share/classes/java/lang/classfile/ClassSignature.java

```diff
@@ -41,7 +41,10 @@ public sealed interface ClassSignature
     /** {@return the type parameters of this class} */
     List<Signature.TypeParam> typeParameters();
 
-    /** {@return the instantiation of the superclass in this signature} */
+    /** {@return the instantiation of the superclass in this signature}
```

Comment:
```suggestion
    /**
     * {@return the instantiation of the superclass in this signature}
```
I think this is our preference if we have multi-line specs.

Reply:
Fixed in [91df97f](https://github.com/openjdk/jdk/pull/18954/commits/91df97f7661774bf4f262260e0732507399bce68), Thanks.